projects/ngrx.io/content/guide/signals/signal-state.md

```diff
@@ -86,7 +86,7 @@ patchState(
 
 <div class="alert is-critical">
 
-Updaters passed to the `patchState` function must perform state updates in an immutable manner.
+The state is frozen during development mode. This means all changes via `patchState` must be immutable. If a mutable change occurs — whether through `patchState`, a component, a service, or any other entity with access to the state — an error will be thrown.
```

Comment:
> The state is frozen during development mode.

We can avoid sharing implementation details and only mention what's important for library consumers. (Example - `strictStateImmutability` desc: https://ngrx.io/guide/store/configuration/runtime-checks#strictstateimmutability)

 ```suggestion
Updaters passed to the `patchState` function must perform state updates in an immutable manner.
If a mutable change occurs to the state object, an error will be thrown in development mode.
```